www/src/context/SettingsContext.tsx

```diff
@@ -44,12 +44,14 @@ function SettingsContextProvider({ children }) {
     }
     setSettings(prevState => ({ ...prevState, [key]: value }));
     global.localStorage.setItem('pgn__settings', JSON.stringify({ ...settings, [key]: value }));
-    global.analytics.track(`${key[0].toUpperCase() + key.slice(1)} change`, { [key]: value });
+    global.analytics.track(`openedx.paragon.docs.settings.${key}.changed`, { [key]: value });
   };
 
   const toggleSettings = (value: boolean) => {
     setShowSettings(value);
-    global.analytics.track('Toggle Settings', { value: value ? 'show' : 'hide' });
+    global.analytics.track(`openedx.paragon.docs.settings.${value ? 'opened' : 'closed'}`, {
+      value: value ? 'show' : 'hide',
```

Comment:
No need to send additional data here because event name already describes whether settings were opened or closed. (right now when settings are opened you send  `'openedx.paragon.docs.settings.opened', { value: 'show' }` event, but it is enough to just send `openedx.paragon.docs.settings.opened` event without additional data)